### PR TITLE
BigQuery: simplify test name

### PR DESCRIPTION
**Pull request by chemelnucfin**
also for testing circleci

### CODE REVIEW

**Comment by chemelnucfin** [comment]
@tswast I'll merge it after I use it to mess with the circleCI issue.

File: bigquery/google/cloud/bigquery/client.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 from google.cloud.bigquery.table import _row_from_mapping
 
 
-_DEFAULT_CHUNKSIZE = 1048576  # 1024 * 1024 B = 1 MB
```
Comment (tswast) [inline bigquery/google/cloud/bigquery/client.py]: Sorry, I don't really understand what's going on in this PR. Marking as needs changes so it doesn't merge.

Comment (chemelnucfin) [reply bigquery/google/cloud/bigquery/client.py]: @tswast Sorry, I was just using that as an example to check circleCI issues.  You are right, I forgot to mark it as Do Not Merge.

Comment (chemelnucfin) [reply bigquery/google/cloud/bigquery/client.py]: @tswast just ignore this PR.  I'll revert it back to just the `snippets_test` to `snippets` later.

**Comment by chemelnucfin** [comment]
@tswast I changed it to what it was when you approved it.

Sorry again for not being clear in my intentions earlier.

**Comment by chemelnucfin** [comment]
merging.  snippets test times out.

**Pull request merged**